### PR TITLE
Changelog entry for v0.13.0

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

This PR adds the changelog entry for v0.13.0. 

- **Release checklist**: #3361
- **Previous changelog PR**: #3201
- **Preview**: https://pygmt-dev--3425.org.readthedocs.build/en/3425/changes.html

**Checklist**

- [x] Add a new entry in `doc/_static/version_switch.js` for documentation switcher
- [x] Add the version to the compatibility table `doc/minversions.md`
- [x] Update citation information `CITATION.cff` and BibTeX at https://github.com/GenericMappingTools/pygmt#citing-pygmt
  - [x] Update authorship list if necessary
  - [x] Update DOI (and URL for BibTeX)
  - [x] Update version
  - [x] Update date released
  - [x] Update the BibTeX key
- [x] Copy the draft release note from https://github.com/GenericMappingTools/pygmt/releases to `doc/changes.md`
- [x] Edit the changelog to look nice ([see maintainers guide for details](https://www.pygmt.org/dev/maintenance.html#updating-the-changelog))
  - [x] Update the release date and the DOI badge in `doc/changes.md`
  - [x] Edit the list of people who contributed to the release, linking to their GitHub account. Sort their names by the number of commits made since the last release (e.g., use `git shortlog HEAD...v0.1.2 -sne`)
  - [x] Edit the change list to remove any trivial changes (updates to the README, typo fixes, CI configuration, etc)
  - [x] Update with any remaining PRs merged before release
  - [x] Add highlights

### CODE REVIEW

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:9]: Just a few ideas for potential highlights:
```suggestion
* Add section "Technical Reference"
* Add tutorial "Draping a dataset on top of a topographic surface"
* Add support and tutorial for using non-ASCII characters
```

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:18]: Perhaps we should add a separate category for breaking changes?

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:73]:
```suggestion
* CI: Test NumPy 2.1 in the GMT Tests workflow ([#3401](https://github.com/GenericMappingTools/pygmt/pull/3401))
```

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:81]:
```suggestion
* Add pre-commit config with pre-commit-hooks and enable pre-commit.ci to enable auto updates of hooks ([#3283](https://github.com/GenericMappingTools/pygmt/pull/3283), [#3414](https://github.com/GenericMappingTools/pygmt/pull/3414))
```

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:67]:
```suggestion
```

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:28]: Move to "Maintanence":
```suggestion
```

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:61]:
```suggestion
* SPEC 0: Bump minimum supported versions to xarray 2022.09 ([#3372](https://github.com/GenericMappingTools/pygmt/pull/3372))
* SPEC 0: Set minimum required version to NumPy 1.24+ ([#3286](https://github.com/GenericMappingTools/pygmt/pull/3286))
```

File: doc/changes.md
Comment (seisman) [reply doc/changes.md]: Thanks. Make some changes in 1b2e2ae793d01a91137264139c866feb9747d021.

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:21]:
```suggestion
* pygmt.x2sys_cross: Refactor to use virtualfiles for output tables [**BREAKING CHANGE**] ([#3182](https://github.com/GenericMappingTools/pygmt/pull/3182))
```

File: doc/changes.md
Comment (yvonnefroehlich) [reply doc/changes.md:21]: Would make those changes more / directly visible to users!

Meaning we have to define a new label "breaking changes" and add it to 
https://github.com/GenericMappingTools/pygmt/blob/6dc8eb09fccb7c554ba4a775715533b33dc84d05/.github/release-drafter.yml#L8-L20

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:10]:
```suggestion
* Add the new documentation section "Technical Reference" and document the supported encodings and fonts
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:22]:
```suggestion
* pygmt.show_versions: Warn about incompatible Ghostscript versions ([#3244](https://github.com/GenericMappingTools/pygmt/pull/3244))
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:56]:
```suggestion
* Simplify the "Minimum Supported Versions" page using MyST customized URL schemes ([#3383](https://github.com/GenericMappingTools/pygmt/pull/3383))
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:60]:
```suggestion
* SPEC 0: Bump minimum supported version to xarray 2022.09 ([#3372](https://github.com/GenericMappingTools/pygmt/pull/3372))
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:61]:
```suggestion
* SPEC 0: Bump minimum supported version to NumPy 1.24+ ([#3286](https://github.com/GenericMappingTools/pygmt/pull/3286))
```

File: doc/changes.md
Comment (weiji14) [reply doc/changes.md:21]: Common changelog uses a prefix like `**Breaking**` or `**<subsystem> (breaking):**` for highlighting breaking changes, see https://github.com/vweevers/common-changelog?tab=readme-ov-file#244-prefixes. Maybe we could follow that instead of adding a new subsection?

File: CITATION.cff
Comment (weiji14) [inline CITATION.cff:78]: Didn't want to bring this up so soon, but since Paul passed away, I'm wondering what the convention was in keeping his name on the citation list? Do we remove his name now, keep his name on for a couple more releases, or remove it after his funding/grants run out? See e.g. https://academia.stackexchange.com/questions/116845/dead-author-ethical-guidelines

File: doc/changes.md
Comment (seisman) [reply doc/changes.md]: Sounds good. Changes in 7d3ac6339f085ad41fa2a1174342a5558dadd46b.

File: CITATION.cff
Comment (seisman) [reply CITATION.cff:78]: Personally, I prefer to keep Paul's name at least in the period of the funding.

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:21]:
```suggestion
* **BREAKING**: pygmt.x2sys_cross: Refactor to use virtualfiles for output tables ([#3182](https://github.com/GenericMappingTools/pygmt/pull/3182))
```

**Comment by seisman** [comment]
This PR is ready for final review. I plan to merge this PR and make the release in 12 hours (before midnight for UTC+8 time zone).

File: doc/changes.md
Comment (weiji14) [inline doc/changes.md:10]:
```suggestion
* Add new documentation section "Technical Reference" and document the supported encodings and fonts
```

File: doc/changes.md
Comment (weiji14) [inline doc/changes.md:21]:
```suggestion
* **Breaking**: pygmt.x2sys_cross: Refactor to use virtualfiles for output tables ([#3182](https://github.com/GenericMappingTools/pygmt/pull/3182))
```

File: doc/changes.md
Comment (weiji14) [inline doc/changes.md:53]: Move this under 'Enhancements'?

File: doc/changes.md
Comment (weiji14) [inline doc/changes.md:61]: These should go under 'Deprecations' I think. We used to classify the NEP29 PRs as 'deprecations' (e.g. at https://www.pygmt.org/v0.12.0/changes.html#release-v0-11-0-2024-02-01), but somehow put the SPEC 0 PRs under 'maintenance' in Release v0.12.0. Probably should move them all to Deprecations.

File: doc/changes.md
Comment (weiji14) [inline doc/changes.md:63]: This might affect some users, put under 'Enhancements'?

File: doc/changes.md
Comment (seisman) [reply doc/changes.md]: Good suggestions and done in 60f3f0b84341a62526bcb72a72751e7a8b1a2501.

File: doc/changes.md
Comment (seisman) [inline doc/changes.md:76]:
```suggestion
* Add pre-commit config with pre-commit-hooks and enable pre-commit.ci to update hooks quarterly ([#3283](https://github.com/GenericMappingTools/pygmt/pull/3283), [#3414](https://github.com/GenericMappingTools/pygmt/pull/3414))
```

**Review by weiji14 — APPROVED** [review]
Looks good to me! Wait for second approval before merge.

File: doc/changes.md
Comment (michaelgrund) [inline doc/changes.md:41]:
```suggestion
* load_tile_map: Register the rio accessor by importing rioxarray, so the returned raster has CRS ([#3323](https://github.com/GenericMappingTools/pygmt/pull/3323))
```

File: doc/changes.md
Comment (michaelgrund) [inline doc/changes.md:43]:
```suggestion
* Replace rio.set_crs with rio.write_crs in the load_tile_map function ([#3321](https://github.com/GenericMappingTools/pygmt/pull/3321))
```

**Review by michaelgrund — APPROVED** [review]
Looks good!

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:61]: Maybe we should move this to the "Documentation" section. 
```suggestion
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:52]:
```suggestion
* Update the onboarding access checklist in Maintainers Guides ([#3404](https://github.com/GenericMappingTools/pygmt/pull/3404))
* Add an offboarding access checklist for maintainers ([#3411](https://github.com/GenericMappingTools/pygmt/pull/3411))
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:26]:
```suggestion
* clib: Refractor to avoid checking GMT version repeatedly and only check once when loading the GMT library ([#3254](https://github.com/GenericMappingTools/pygmt/pull/3254))
```

File: doc/changes.md
Comment (yvonnefroehlich) [inline doc/changes.md:62]:
```suggestion
* sphinx-gallery: Temporarily pin to < 0.17.0 ([#3350](https://github.com/GenericMappingTools/pygmt/pull/3350))
```

**Review by yvonnefroehlich — APPROVED** [review]
Looks good for me!

File: doc/changes.md
Comment (yvonnefroehlich) [reply doc/changes.md]: Sorry, just saw there is a typo in my review, it's Refactor, no 'r'.

**Pull request merged**
